owasp-top10-2017-apps/a9/cimentech/deployments/docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
     ports:
       - 80:80
     volumes:
-      - ../app/html:/var/www/html
+      - ../app/html/sites/:/var/www/html/sites
```

Comment:
Cool! We have a custom `robots.txt` (https://github.com/globocom/secDevLabs/blob/master/owasp-top10-2017-apps/a9/cimentech/app/html/robots.txt)  that appears in the readme: 

![image](https://user-images.githubusercontent.com/11429002/116296274-65bd1b80-a770-11eb-8b76-70001cd6ba8d.png)

Can we find a way to keep that?